indico/modules/events/reminders/forms.py

```diff
@@ -64,8 +67,6 @@ def __init__(self, *args, **kwargs):
         self.reply_to_address.choices = (self.event
                                          .get_allowed_sender_emails(extra=self.reply_to_address.object_data)
                                          .items())
-        if self.event.type_ == EventType.lecture:
```

Comment:
I think this should stay now that there's the separate option. The summary is still useless for lectures.

Reply:
Agreed.

Reply:
Done